### PR TITLE
poll: pass real data to react

### PR DESCRIPTION
**Pull request by xi**
this replaces the sample data with real data from the model. Note that `object` in the poll detail view is currently not defined, so it does not yet work.

### CODE REVIEW

File: apps/polls/assets/Polls.jsx [hunk 1]
```diff
@@ -6,18 +6,9 @@ var Poll = React.createClass({
...
-        count: 22434
-      }, {
-        label: 'Nein',
-        count: 40062
-      }, {
-        label: 'Vielleicht',
-        count: 17627
-      }],
-      ownVote: null,
+      title: this.props.poll.title,
```
Comment (kleingeist) [inline apps/polls/assets/Polls.jsx]: Disclaimer: this question is more to understand react then a review.

Why are title and choices (at least the labels) state values? from the tutorials i would assume they should stay in the props. Unfortunately the choice.count and choice.ownChoice properties are state as they will change on  a vote :(

Comment (vellip) [inline apps/polls/assets/Polls.jsx]: As we discussed, see https://shripadk.github.io/react/tips/initial-ajax.html. We will keep this pattern for consistency now though.
